service/javadsl/server/src/main/scala/com/lightbend/lagom/internal/javadsl/server/ServiceInfoProvider.scala

```diff
@@ -12,7 +12,7 @@ import com.lightbend.lagom.javadsl.api.ServiceInfo
  */
 @Singleton
 class ServiceInfoProvider(primaryServiceInterface: Class[_], secondaryServices: Array[Class[_]]) extends Provider[ServiceInfo] {
-  @Inject private var serverBuilder: JavadslServerBuilder = _
+  @Inject private val serverBuilder: JavadslServerBuilder = _
```

Comment:
`serverBuilder` must be mutable, since it gets injected by Guice after instantiation.